src/zenml/models/v2/misc/full_stack.py

```diff
@@ -13,15 +13,20 @@
 #  permissions and limitations under the License.
 """Models representing full stack requests."""
 
-from typing import Any, Dict, List, Optional, Union
-from uuid import UUID
+from typing import TYPE_CHECKING, Any, Dict, List, Optional
```

Comment:
I wasn't exactly sure what to do with those actually.

As I mentioned in my TODO comment down below, I feel like this could be cleaned up even further. When you take a look, each `Info` class sort of represents the request model of the corresponding type. However, since the main goal was to unify the stack creation endpoints, I haven't touched those yet and left them where they were.

If you have any recommendations, I would be more than happy to implement them though. 😄

Reply:
I think I'd leave them as is, maybe once our request models don't have the user and workspace ID anymore we can merge them. Just the file seems wrong, they should just be in the file where the stack request is defined IMO

Reply:
I get the pain and I would love to merge them as well when the time comes. 

Usually, I thought most of these helper models (which do not fit the `base-request-update-body-metadata-resource-response-filter` paradigm) were in `misc` folder. Happy to move them though.

Reply:
Can also keep them in that folder, but `full_stack` is not a thing anymore and never should have been. If we rename the file to something that makes sense, I'm happy.

Reply:
I renamed it to `info_models`. Hope that makes more sense.